erpnext/patches.txt

```diff
@@ -658,3 +658,4 @@ erpnext.patches.v12_0.rename_mws_settings_fields
 erpnext.patches.v12_0.set_correct_status_for_expense_claim
 erpnext.patches.v12_0.set_updated_purpose_in_pick_list
 erpnext.patches.v12_0.repost_stock_ledger_entries_for_target_warehouse
+erpnext.patches.v12_0.update_end_date_and_status_in_email
```

Comment:
Can we change the patch name from "update_end_date_and_status_in_email" to "update_end_date_and_status_in_email_campaign"?